selfdrive/controls/controlsd.py

```diff
@@ -203,70 +209,87 @@ def update_events(self, CS):
       self.events.add(EventName.controlsInitializing)
       return
 
-    # Create events for battery, temperature, disk space, and memory
-    if EON and (self.sm['peripheralState'].pandaType != PandaType.uno) and \
-       self.sm['deviceState'].batteryPercent < 1 and self.sm['deviceState'].chargingError:
-      # at zero percent battery, while discharging, OP should not allowed
-      self.events.add(EventName.lowBattery)
-    if self.sm['deviceState'].thermalStatus >= ThermalStatus.red:
-      self.events.add(EventName.overheat)
-    if self.sm['deviceState'].freeSpacePercent < 7 and not SIMULATION:
```

Comment:
```
 results for segment bd6a637565e91581|2021-10-30--15-14-53--2 *****
	controlsd
		logs are not same length: 23990 VS 23989
		Counter({'sendcan': 5978, 'controlsState': 5978, 'carState': 5978, 'carControl': 5978, 'carEvents': 76, 'carParams': 2})
		Counter({'sendcan': 5978, 'controlsState': 5978, 'carState': 5978, 'carControl': 5978, 'carEvents': 75, 'carParams': 2})
	radard
	plannerd
	calibrationd
	dmonitoringd
	locationd
	paramsd
	ubloxd

TEST FAILED
```
I'm not sure if this failure was caused by this line. the EventName.lowBattery will be added to events if deviceState has not been received on startup